release.sh

```diff
@@ -2,32 +2,61 @@
 
 set -e
 
-last_tag=$(git tag --sort=-creatordate | head -n 1)
+last_tag=$(git tag --sort=-creatordate| head -n 1)
 new_tag=$(semver bump patch "$last_tag")
 timestamp=$(date -u +%Y%m%d%H%M%S)
 ref=$(echo "$BRANCH" | sed -e "s/[^a-zA-Z0-9]//g")
 new_version=$(if [ "$ref" = "master" ]; then echo "${new_tag}"; else echo "${new_tag}-${ref}.$timestamp"; fi)
 
-echo "Bump version: ${last_tag} -> ${new_version}"
-git tag "${new_version}"
-
-# Create image tag
-image_tag="${IMAGE_NAME}:${new_version}"
-docker tag "${IMAGE_NAME}" "${image_tag}"
-
-echo "Tagged docker image $image_tag"
-
-# If the branch is master, publish it.
-if [ "$ref" = "master" ]; then
+validateFileChanges() {
+  # Define a list of sensible files.
+  sensible_files=(Dockerfile .dockerignore requirements.txt)
+
+  # Validate if sensible file are in list.
+  for Item in ${sensible_files[*]} ;
```

Comment:
Is there any reason you are using `Item` instead of smaller case `item`? I feel the name `file` is a better variable here.

```bash
for file in ${sensible_files[*]};
do
  ...
```